include/ignition/math/Helpers.hh

```diff
@@ -893,6 +893,50 @@ namespace ignition
                                                 // .000 - 0.999
 
 
+    /// \brief Check if the given string represents a time.
+    /// An example time string is "0 00:00:00.000", which has the format
+    /// "DAYS HOURS:MINUTES:SECONDS.NANOSECONDS"
```

Comment:
I see `NANOSECONDS` in this comment but milliseconds mentioned in the `time_regex` documentation. Should this change to `MILLISECONDS`?

Reply:
Yup, thanks. Fixed.